api/Node.json

```diff
@@ -165,10 +165,10 @@
               "version_added": null
             },
             "firefox": {
-              "version_added": null
+              "version_added": false
```

Comment:
Seems like https://developer.mozilla.org/docs/Web/API/Node/baseURIObject is a XUL thing and we should remove the entry.

Reply:
I've sent https://github.com/mdn/browser-compat-data/pull/7520 which would conflict.